pkg/server/login/login.go

```diff
@@ -166,25 +169,29 @@ func (l *Login) handleLogin(w http.ResponseWriter, req *http.Request) {
 		failed(errorCodeAccessDenied, w, req)
 		return
 	}
-	result := metrics.SuccessResult
-	defer func() {
-		metrics.RecordFormPasswordAuth(result)
-	}()
+
+	audit.AddUsername(req, username)
```

Comment:
It was not easy to do tests with a `roundtripper` in `login_test.go`, so I ditched it as I would need to re-write the testing structure as as whole.